build.sbt

```diff
@@ -89,6 +89,8 @@ lazy val core = (project in file("core"))
     ),
 
     javaOptions += "-Xmx1024m",
+    javacOptions ++= Seq("-source", "1.8"),
+    Compile / compile / javacOptions ++= Seq("-target", "1.8"),
```

Comment:
What's the difference between these two lines?

Could you move `scalacOptions` and `javacOptions` to `commonSettings` and remove them from each individual project?

Reply:
`-target 1.8` can not be passed as a parameter to `javadoc`. By restricting the option to the `Compile / compile` task `javadoc` will still work. I got the workaround from https://github.com/sbt/sbt/issues/355

Reply:
Thanks! I've moved the `scalacOptions` and `javacOptions` options to `commonSettings`. I took the liberty to move the `scalacOptions` specifying `target:jvm-1.8` as well.